### PR TITLE
Maks Servatiak

### PR DESCRIPTION
**Pull request by MaksServatyak**

### CODE REVIEW

File: src/Lesson4/Array.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package Lesson4;
+
+import java.util.Random;
+
+public class Array {
+    public int[] arr;
+
```
Comment (Volodymyr95) [inline src/Lesson4/Array.java:7]: where constructor for pass array?

**Review by Volodymyr95 — APPROVED** [review]
Good Job!

**Review by AndrewRorat — APPROVED** [review]

File: src/Homework5/CharArray.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package Homework5;
+
+public class CharArray extends Array{
```
Comment (Volodymyr95) [inline src/Homework5/CharArray.java:3]: where all methods?

**Review by natalia-kozak — APPROVED** [review]

**Review by natalia-kozak — APPROVED** [review]
Why didn`t you call all methods from others classes in class Main?
What about methods searching by id, by country and by COD? You didn't call them in class main.

**Pull request merged**